include/bpf/libbpf.h

```diff
@@ -731,6 +731,77 @@ bpf_program__unload(struct bpf_program* prog);
 int
 bpf_program__unpin(struct bpf_program* prog, const char* path);
 
+/**
+ * @brief Attach an XDP program to a given interface.
+ *
+ * @param[in] ifindex The interface index to attach to, or -1 to detach.
+ * @param[in] fd File descriptor of program to attach.
+ * @param[in] flags Flags (currently 0).
```

Comment:
I think we should update the description for `Flags` to mention about `XDP_FLAGS_REPLACE`.

Reply:
done